src/components/MovieCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import '../stylesheets/moviecard.css'
+
+const baseURL = 'https://image.tmdb.org/t/p'; // 
+const posterSize = '/w500'
+
+export default function MovieCard({movie}){
+
+  const {title, vote_average, poster_path} = movie; //destructuring movie prop (object) and storing info we need in variables
```

Comment:
React doesn't use snake case, let's change the variable names to camel case
```suggestion
  const {title, voteAverage, posterPath} = movie; //destructuring movie prop (object) and storing info we need in variables
```

Reply:
For this correct me if I am wrong but I did vote_average and poster_path because I am destructing the movie object which has fields vote_average and poster_path. If I change those to camel case it wont destructure properly